packages/data-extensions/src/api.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * Union of supported API versions
+ */
+export type ApiVersion = 'unstable';
+
+export interface Auth {
+  /**
+   * Retrieves a Shopify OpenID Connect ID token for the current user.
+   */
+  idToken: () => Promise<string | null>;
+}
+
+/**
+ * GraphQL error returned by the Shopify Admin API.
+ */
+export interface GraphQLError {
```

Comment:
We couldn't find a way to share these, eh?

Reply:
I just pushed up another commit with the changes to make these APIs shareable. It did require changing the `rootDir` to a shared one a level up. I used it in the `ui-extensions` and `data-extensions` standard API definitions, but I was hesitant to make updates throughout checkout to use the shared APIs. Maybe that can be a follow up?

Reply:
I fought with it for awhile (changed `rootDir`, added `paths`, `includes`, etc. and reorganized the exports) and I thought I had everything building locally, but that broke the other packages when it got to CI. What do you think about making this a follow up clean up task?

Reply:
Yeah, forget it. I was just wondering if it was possible